src/styles/catalog.scss

```diff
@@ -0,0 +1,105 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  max-width: catalogWidth($cardWidth, $gap);
+}
+
+.card {
+  margin-top: 49.5px;
```

Comment:
card can't have outer spacing, add mix class with margin
https://www.youtube.com/watch?v=JOJ-mENrg_s

Reply:
also, let's add for each card mix class margin-bottom and margin-top for whole catalog (also with mix)